apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -697,7 +697,11 @@ public void loadSyntheticData() {
    *
    * <p>This test ensures that changes to RifLoader that break compatibility with Synthea-generated
    * data are flagged.
+   *
+   * <p>Note: For (https://jira.cms.gov/browse/BFD-2222) this test will be temporarily disabled and
+   * re-enabled them once Synthea is updated for the new fields
    */
+  @Disabled
```

Comment:
Last synthea meeting MITRE mentioned they were going to merge the changes; is it perhaps merged by now (and these can be re-enabled?)

Reply:
reenabling